pandas/core/frame.py

```diff
@@ -159,7 +159,7 @@
 
 from pandas.io.common import get_handle
 from pandas.io.formats import console, format as fmt
-from pandas.io.formats.info import DataFrameInfo
+from pandas.io.formats.info import BaseInfo, DataFrameInfo
```

Comment:
do we need to add the import of BaseInfo just for the docstring? could the docstring from BaseInfo.render be added to DataFrameInfo.render (could be a follow-on)

Reply:
Yes, I plan to fix it in the follow-up.
I struggled with ``doc`` decorator, so I could not figure out how to make it work properly so far.
I will create another PR on this and would be very grateful if you would suggest there how to resolve the outstanding issues.